bodhi/server/scripts/skopeo_lite.py

```diff
@@ -692,7 +692,11 @@ def _copy_blob(self, digest, size):
         elif isinstance(self.src, RegistryEndpoint) and isinstance(self.dest, RegistryEndpoint):
             if self.src.registry == self.dest.registry:
                 self.dest.link_blob(digest, self.src.repo)
-        # Other forms of copying are not needed currently, and not implemented
+            else:  # pragma: no cover
```

Comment:
I'm not sure if these kind of changes should be done in PR, that is increasing the coverage.

Reply:
See my own comment there: "I'm not sure about introducing pragma: no cover here because we don't use it anywhere else. Alternatively, I could replace the conditions that can't fail with asserts." What do you think?

Reply:
Current push has the asserts instead.